src/CentreonUser/Tests/Application/Webservice/TimeperiodWebserviceTest.php

```diff
@@ -62,6 +69,7 @@ protected function setUp(): void
 
     /**
      * Test the method getList
+     * @return void
      */
     public function testGetList()
```

Comment:
```suggestion
    public function testGetList(): void
```

return void isnt necessary in phpdoc, but should be typehinted in signature

Reply:
i haven't commented the other @return void, i let you handle them ^^

Reply:
Done